client/tx/aux_builder_test.go

```diff
@@ -200,7 +199,7 @@ func TestAuxTxBuilder(t *testing.T) {
 	for _, tc := range testcases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			b = tx.NewAuxTxBuilder()
+			b = NewAuxTxBuilder()
```

Comment:
was there a reason to change the package name?

Reply:
I think it for testing their private function `getSimSignatureData` and set `fromName` on the Factory that is usually private. For the latter, they could use clientCtx with a from name however.